core/mondoo-kubernetes-security.mql.yaml

```diff
@@ -5624,17 +5161,8 @@ queries:
     title: ReplicaSets should not run with NET_RAW capability
     impact: 80
     mql: |
-      k8s.replicaset {
-        podSpec['containers'] {
-          _['securityContext']['capabilities'] != null
-          _['securityContext']['capabilities'] {
-            _['add'] == null || _['add'].none(_.upcase == "ALL")
-            _['add'] == null || _['add'].none(_.upcase == "NET_RAW")
-            _['drop'] != null
-            _['drop'].any(_.upcase == "NET_RAW") || _['drop'].any(_.upcase == "ALL")
-          }
-        }
-      }
+      k8s.replicaset.podSpec['containers'].all( _['securityContext']['capabilities'] { _['add'].none( _ == /^NET_RAW$|^ALL$/ ) } )
+      k8s.replicaset.podSpec['containers'].all( _['securityContext']['capabilities'] { _['drop'].any( _ == /^NET_RAW$|^ALL$/ ) } )
```

Comment:
Same as L5057 - L5058:

- `.upcase` and `['add'] == null`

Reply:
@mm-weber does this also need `_['drop'] == null`